HW-PHP-2022-12-04/task-1.php

```diff
@@ -0,0 +1,8 @@
+<?php
+//Разработайте функцию для вычисления суммы произвольного количества заданных элементов.
+$str = (string) readline ("Enter elements: ");
+$arr = explode(' ', $str);
+function sum ($arr) {
```

Comment:
```suggestion
function sum (...$arr) {
```
произвольное кол-во параметров можно определить так